test/test_other.py

```diff
@@ -823,7 +823,6 @@ def test_em_config_missing_arg(self):
     out = self.expect_fail([EMCC, '--em-config'])
     self.assertContained('error: --em-config must be followed by a filename', out)
 
-  @disabled('Updating underlying tool output')
```

Comment:
How is it that this test can be re-enabled without any changes? I thought it was disabled so that something roll in, but that suggests the test errored on those things and needed some change to fix that, but maybe I'm confused.

Reply:
It looks like expectations file for this test (`test/other/test_emsize.out`) was updated.

Reply:
Yes, the changes to test_emsize.out make this test pass again.

Reply:
Thanks, sorry, I misread the diff.